django/install.sls

```diff
@@ -1,10 +1,62 @@
 {% from "django/map.jinja" import django with context %}
+{% set app_name = django.get('app_name', 'django_app') %}
 
-include:
-  - .service
+create_django_app_user:
+  user.present:
+    - name: {{ django.user }}
+    - createhome: False
 
-django:
+django_system_dependencies:
   pkg.installed:
     - pkgs: {{ django.pkgs }}
-    - require_in:
-        - service: django_service_running
+
+create_deployment_target_directory:
+  file.directory:
+    - name: /opt/{{ app_name }}
+    - user: {{ django.user }}
+    - group: {{ django.group }}
+    - recurse:
+        - user
+        - group
+
+{% set app_source = django.app_source %}
```

Comment:
What is `django.app_source`?

Reply:
It is a pillar attribute that gets merged into the map.jinja datastructure.